assets/stylesheets/_components.scss

```diff
@@ -419,6 +419,7 @@
 @import 'my-sites/stats/stats-site-overview/style';
 @import 'my-sites/stats/stats-tabs/style';
 @import 'my-sites/stats/stats-views/style';
+@import 'my-sites/stats/wordads/style';
```

Comment:
This can just be included from the wordads component with `import 'style.scss';`. We're trying to move off of the global imports.